src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,30 @@
-export const PostInfo = () => (
-  <>Put the post here</>
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">
+        {`${post.title}`}
+      </h3>
+
+      <UserInfo user={post.user} />
+    </div>
+
+    <p className="PostInfo__body">
+      {`${post.body}`}
+    </p>
+
+    {post.comments.length === 0
```

Comment:
you can write just `post.comments.length` 
this variable will automatically be converted to a boolean value